tasks/seq/morozov_e_lineare_image_filtering_block_gaussian/func_tests/main.cpp

```diff
@@ -0,0 +1,211 @@
+#include <gtest/gtest.h>
+
+#include <cstdint>
+#include <memory>
+#include <vector>
+
+#include "core/task/include/task.hpp"
+#include "seq/morozov_e_lineare_image_filtering_block_gaussian/include/ops_seq.hpp"
+
+TEST(morozov_e_lineare_image_filtering_block_gaussian, empty_image_test) {
+  int n = 0;
+  int m = 0;
+  std::vector<int> image(n * m, 1);
+  std::vector<int> image_res(n * m, 1);
+
+  // Create task_data
+  auto task_data_seq = std::make_shared<ppc::core::TaskData>();
+  task_data_seq->inputs.emplace_back(reinterpret_cast<uint8_t *>(image.data()));
+  task_data_seq->inputs_count.emplace_back(n);
+  task_data_seq->inputs_count.emplace_back(m);
+  task_data_seq->outputs.emplace_back(reinterpret_cast<uint8_t *>(image_res.data()));
+  task_data_seq->outputs_count.emplace_back(n);
+  task_data_seq->outputs_count.emplace_back(m);
+
+  // Create Task
+  morozov_e_lineare_image_filtering_block_gaussian::TestTaskSequential test_task_sequential(task_data_seq);
+  ASSERT_EQ(test_task_sequential.Validation(), false);
+}
+TEST(morozov_e_lineare_image_filtering_block_gaussian, size_input_not_equal_size_output_test1) {
+  int n = 0;
+  int m = 0;
+  std::vector<int> image(n * m, 1);
+  std::vector<int> image_res((n + 1) * m, 1);
+
+  // Create task_data
+  auto task_data_seq = std::make_shared<ppc::core::TaskData>();
+  task_data_seq->inputs.emplace_back(reinterpret_cast<uint8_t *>(image.data()));
+  task_data_seq->inputs_count.emplace_back(n);
+  task_data_seq->inputs_count.emplace_back(m);
+  task_data_seq->outputs.emplace_back(reinterpret_cast<uint8_t *>(image_res.data()));
+  task_data_seq->outputs_count.emplace_back(n);
+  task_data_seq->outputs_count.emplace_back(m);
+
+  // Create Task
+  morozov_e_lineare_image_filtering_block_gaussian::TestTaskSequential test_task_sequential(task_data_seq);
+  ASSERT_EQ(test_task_sequential.Validation(), false);
+}
+TEST(morozov_e_lineare_image_filtering_block_gaussian, size_input_not_equal_size_output_test2) {
+  int n = 0;
+  int m = 0;
+  std::vector<int> image(n * m, 0);
+  std::vector<int> image_res(n * m, 0);
+
+  // Create task_data
+  auto task_data_seq = std::make_shared<ppc::core::TaskData>();
+  task_data_seq->inputs.emplace_back(reinterpret_cast<uint8_t *>(image.data()));
+  task_data_seq->inputs_count.emplace_back(n);
+  task_data_seq->inputs_count.emplace_back(m);
+  task_data_seq->outputs.emplace_back(reinterpret_cast<uint8_t *>(image_res.data()));
+  task_data_seq->outputs_count.emplace_back(n);
+  task_data_seq->outputs_count.emplace_back(m);
+
+  // Create Task
+  morozov_e_lineare_image_filtering_block_gaussian::TestTaskSequential test_task_sequential(task_data_seq);
+  ASSERT_EQ(test_task_sequential.Validation(), false);
+}
+TEST(morozov_e_lineare_image_filtering_block_gaussian, main_test1) {
+  int n = 5;
+  int m = 5;
+  std::vector<double> image = {1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1};
```

Comment:
Its hard to read in this form, you can write like this:
```suggestion
  int n = 5;
  int m = 5;
  // clang-format off
  std::vector<double> image = 
  {1, 1, 1, 1, 1,
   1, 1, 1, 1, 1,
   1, 1, 1, 1, 1,
   1, 1, 1, 1, 1,
   1, 1, 1, 1, 1};
   // clang-format on
```

Reply:
fixed